draft-ietf-jose-json-proof-algorithms.md

```diff
@@ -219,256 +219,79 @@ See the example in the appendix of the JSON Web Proof draft.
+Figure: Issued JWP (JSON serialization)
 
-The verifier decodes the JWP issuer protected header and payload values into a messages array, skipping any non-revealed payloads.  The current BBS implementation embeds the revealed indices into the output proof value, so the verification messages array only needs to include the disclosed messages.
+<{{./fixtures/build/bbs-issuer.compact.jwp}}
+Figure: Issued JWP (compact serialization)
 
-In the implementation, the method used to verify the proof is `blsVerifyProof({verifyProof, publicKey, [issuer_header, payload2, ...], presentation_header)`.
+### Issuance proof verification
 
-### JPA Registration
+Holder verification of the signature on issuance form is performed using the `Verify` operation from [@!I-D.irtf-cfrg-bbs-signatures, section 3.4.2].
 
-Proposed JWP `alg` value for BBS based on the software implementation is "BBS-X".
-
-### Example
-
-The following example uses the given BLS12-384 key-pair:
-
-Public:
-```json
-[179, 209, 122, 60, 230, 37, 188, 86, 19, 19, 4, 36, 240, 230, 79,
-178, 230, 147, 9, 60, 239, 41, 233, 167, 190, 252, 154, 35, 39, 201,
-238, 73, 77, 228, 20, 47, 109, 174, 15, 168, 187, 145, 126, 85, 83,
-151, 48, 30, 13, 237, 92, 179, 124, 181, 211, 204, 187, 222, 229,
-234, 182, 94, 60, 157, 19, 148, 162, 48, 185, 134, 177, 168, 68,
-115, 167, 48, 92, 181, 168, 53, 52, 246, 201, 112, 103, 23, 159,
-138, 225, 13, 165, 171, 251, 112, 163, 96]
-```
-Figure: bbs-issuer-public-octets
+This operation utilizes the issuer's public key as `PK`, the proof as `signature`, the protected header octets as `header` and the array of payload octets as `messages`).
 
-Private:
-```json
-[72, 125, 227, 97, 150, 148, 186, 145, 110, 46, 135, 232, 104, 204,
-128, 242, 73, 151, 72, 162, 0, 54, 139, 146, 221, 137, 34, 74, 1,
-42, 140, 206]
-```
-Figure: bbs-issuer-private-octets
-
-The protected header used is:
-```json
-{
-  "iss": "https://issuer.example",
-  "claims": [
-    "family_name",
-    "given_name",
-    "email",
-    "age"
-  ],
-  "typ": "JPT",
-  "alg": "BBS-X"
-}
-```
-Figure: bbs-issuer-protected-header
-
-The first payload is the string `"Doe"` with the octet sequence of `[ 34, 68, 111, 101, 34 ]` and base64url-encoded as `IkRvZSI`.
+### Presentation
 
-The second payload is the string `"Jay"` with the octet sequence of `[ 34, 74, 97, 121, 34 ]` and base64url-encoded as `IkpheSI`.
+Derivation of a presentation is done by the holder using the `ProofGen` operation from [@!I-D.irtf-cfrg-bbs-signatures, section 3.4.3].
 
-The third payload is the string `"jaydoe@example.org"` with the octet sequence of `[ 34, 106, 97, 121, 100, 111, 101, 64, 101, 120, 97, 109, 112, 108, 101, 46, 111, 114, 103, 34 ]` and base64url-encoded as `ImpheWRvZUBleGFtcGxlLm9yZyI`.
+This operation utilizes the issuer's public key as `PK`, the issuer protected header as `header`, the issuance proof as `signature`, the issuance payloads as `messages`, and the holder's presentation protected header as `ph`.
```

Comment:
It seems inconsistent that `PK` is capitalized and `ph` is not.  Is there a reason that we shouldn't capitalize `PH`?  Or are these identifiers from other specs, in which case we should use them as-is?

Reply:
That is the capitalization within the BBS Signatures spec

Reply:
OK - then leave it that way.  Thanks.